app/pages/system/silos/SiloIpPoolsTab.tsx

```diff
@@ -142,19 +143,22 @@ export function SiloIpPoolsTab() {
     },
   ]
 
+  const resourceInformation = (
+    <>
+      Users in this silo can allocate external IPs from these pools for their instances. A
+      silo can have at most one default pool. IPs are allocated from the default pool when
+      users ask for one without specifying a pool. Read the docs to learn more about{' '}
+      <ExternalLink href={links.ipPoolsDocs}>managing IP pools</ExternalLink>.
+    </>
+  )
```

Comment:
I think the need or least the temptation to pull this out to make it look clean is a point against the props approach. Compare to

```tsx
<AboveTable>
  <AboveTable.Text>
    Users in this silo can allocate external IPs from these pools for their instances.
    A silo can have at most one default pool. IPs are allocated from the default pool
    when users ask for one without specifying a pool. Read the docs to learn more
    about <ExternalLink href={links.ipPoolsDocs}>managing IP pools</ExternalLink>.
  </AboveTable.Text>
  <AboveTable.Button onClick={() => setShowLinkModal(true)}>
    Link pool
  </AboveTable.Button>
</AboveTable>
```

And all the definitions get small and simple:

```tsx
const AboveTable = classed.div`mb-4 flex items-end justify-between space-x-8`
AboveTable.Text = classed.p`max-w-2xl text-sans-md text-secondary`
AboveTable.Button = (props: ButtonProps) => <Button size="sm" className="shrink-0" {...props} />
AboveTable.Link = (props: LinkProps) => <Link className={buttonStyle({ size: 'sm' }) {...props} />
```